test/spec/components/FeelTemplating.spec.js

```diff
@@ -0,0 +1,608 @@
+import { act } from 'preact/test-utils';
+
+import { waitFor } from '@testing-library/dom';
+import {
+  render
+} from '@testing-library/preact/pure';
+
+import TestContainer from 'mocha-test-container-support';
+
+import {
+  classes as domClasses,
+  query as domQuery
+} from 'min-dom';
+
+import EventBus from 'diagram-js/lib/core/EventBus';
+
+import {
+  expectNoViolations,
+  insertCoreStyles
+} from 'test/TestHelper';
+
+import {
+  DescriptionContext,
+  ErrorsContext,
+  EventContext,
+  PropertiesPanelContext
+} from 'src/context';
+
+import FeelTemplatingEntry, { isEdited } from 'src/components/entries/templating/FeelTemplating.js';
+
+insertCoreStyles();
+
+const noop = () => {};
+
+describe('<FeelTemplating>', function() {
+
+  let container;
+  let clock;
+
+  beforeEach(function() {
+    container = TestContainer.get(this);
+    clock = sinon.useFakeTimers();
+  });
+
+  afterEach(function() {
+    clock.restore();
+  });
+
+  describe('Templating Editor', function() {
+
+    it('should render', function() {
+
+      // given
+      const result = createTemplatingEntry({ container });
+
+      // then
+      expect(domQuery('.bio-properties-panel-entry > .bio-properties-panel-feelers > .bio-properties-panel-feelers-input', result.container)).to.exist;
+    });
+
+
+    it('should update', async function() {
+
+      // given
+      const updateSpy = sinon.spy();
+
+      const result = createTemplatingEntry({ container, setValue: updateSpy });
+
+      const input = domQuery('[role="textbox"]', result.container);
+
+      // when
+      await act(() => input.textContent = 'Template {{x}}');
```

Comment:
Maybe this is not enough to make a proper update? In bpmn-js-properties-panel and form-js we use a helper to do the updates

```js
async function setEditorValue(editor, value) {
  await act(() => {
    editor.textContent = value;
  });

  // Requires 2 ticks to propagate the change
  await act(() => {});
}
```

Not sure whether this is causing the test errors though

Reply:
I pushed a fix. Probs to @marstamm finding this via https://github.com/bpmn-io/properties-panel/pull/225/commits/a4706eff443af23972719a10d75a35e6f99b0aff